src/views/__snapshots__/ceo-view.test.js.snap

```diff
@@ -2,12 +2,17 @@
 
 exports[`CeoName component 1`] = `
 <DocumentFragment>
-  <div>
+  <div
+    class="css-1tgedd4-CeoName"
+  >
     <div>
-      <form>
+      <form
+        class="css-xctk3t-InputNameCeo"
+      >
         <input
+          class="css-lrjsfw-InputNameCeo"
           name="ceo"
-          placeholder="Ceo's name..."
+          placeholder="Ceo's Name"
           type="text"
```

Comment:
This `input`needs a `required` property. We cannot allow the user to bypass it or leave it empty. Or at least set a default name.

Reply:
That is a great point, I am going to create an issue with all this refactoring things.